llvm/include/llvm/SYCLLowerIR/HostPipes.h

```diff
@@ -0,0 +1,34 @@
+//===------- HostPipes.h - get required into about SYCL Host Pipes --------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// The file contains a number of functions to extract corresponding attributes
+// of host pipe variables and save them as a property set for the runtime.
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include "llvm/ADT/MapVector.h"
+
+#include <cstdint>
+#include <vector>
+
+namespace llvm {
+
+class GlobalVariable;
+class Module;
+class StringRef;
```

Comment:
Minor comment: These 2 lines seem redundant.
E.g. HostPipes.cpp uses 'StringRef', but it also has "#include "llvm/ADT/StringRef.h""

Also, "include <vector>" above at line 18 also redundant here. There is nothing in this header file using vector.